modules/claims_api/lib/auth_token_service/base.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module AuthTokenService
+    class Base
+      TOKEN_SERVICE_NAMES = %(bds_ccg)
+
```

Comment:
my original intent was to combine this. so add items like this in the token_service and we wouldn't need 2 files.

Reply:
makes sense, updating it now